src/_variables.scss

```diff
@@ -42,3 +42,17 @@ $c2lc-action-panel-height: 3rem;
 $c2lc-scene-border-width: 0.1rem;
 $c2lc-scene-border-color: #CCE6E2;
 $c2lc-scene-border-style: solid;
+
+/* mixins */
+
+@mixin updatedProgramAnimation($color, $commandName, $theme) {
```

Comment:
The mixin is really an impressive addition, I think we'll be able to clean up a lot of duplicate code in the themes when we're ready.